.github/workflows/android-release.yml

```diff
@@ -0,0 +1,83 @@
+#This is a workflow that creates, downloads, uploads and releases APK-file.
+name: APK Workflow
+
+#Event triggers
+#execute the jobs everytime when a pull_request to the main-branch is created
+on:
+  push:
+    branches: main
+
+jobs:
+  runtests:
```

Comment:
Do we really need to run the tests again. Because in dev they were already run. See comment.

Reply:
This part includes java checkstyle. As we tested before, there are checkstyle errors that we should resolve.